docs/tutorials/control/flutter-app.md

```diff
@@ -0,0 +1,732 @@
+   ```yaml {class="line-numbers linkable-line-numbers"}
+   include: package:flutter_lints/flutter.yaml
+
+   linter:
+     rules:
+       avoid_print: false
+       prefer_const_constructors_in_immutables: false
+       prefer_const_constructors: false
+       prefer_const_literals_to_create_immutables: false
+       prefer_final_fields: false
+       unnecessary_breaks: true
+       use_key_in_widget_constructors: false
+   ```
+
+### Configure iOS-specific code
+
+Now you'll update some configurations in the iOS-specific code to support the [Viam Flutter SDK](https://flutter.viam.dev/).
+
+1. Open <file>ios/Podfile</file>.
+   If Podfile does not exist in that directory, you can generate it by running `flutter pub get` in the root directory of your app.
+
+   At the top of the file you will see the following lines:
+
+   ```{class="line-numbers linkable-line-numbers"}
+   # Uncomment this line to define a global platform for your project
+   # platform :ios, '11.0'
+   ```
+
+   Change that to this:
+
+   ```{class="line-numbers linkable-line-numbers"}
+   # Uncomment this line to define a global platform for your project
+   platform :ios, '13.0'
+   ```
+
+2. Open <file>ios/Runner/Info.plist</file>.
+   It will look something like this:
+
+   {{<imgproc src="/tutorials/flutter-app/info-plist.png" resize="1200x" style="max-width:900px" declared-dimensions="true" alt="The info.plist file open in VS Code. The second line includes 'doctype plist public' and an Apple URL. The fourth line is a dict tag. The fifth line is a key, and subsequent lines contain keys, strings and arrays.">}}
+
+3. Insert this code into the first line after the `<dict>`:
+
+   ```{class="line-numbers linkable-line-numbers"}
+     <key>NSLocalNetworkUsageDescription</key>
+     <string>Smart Machine App requires access to your device's local network to connect to your devices.</string>
+     <key>NSBonjourServices</key>
+     <array>
+         <string>_rpc._tcp</string>
+     </array>
+   ```
+
+   {{<imgproc src="/tutorials/flutter-app/info-plist-insert.png" resize="1200x" style="max-width:900px" declared-dimensions="true" alt="The info.plist file open in VS Code. An arrow labeled 'insert here' indicates that you should insert the code right after the fourth line in the file.">}}
+
+   It should look like this:
+
+   {{<imgproc src="/tutorials/flutter-app/info-plist-pasted.png" resize="1200x" style="max-width:900px" declared-dimensions="true" alt="The resulting file. After the non-indented <dict> tag on the fourth line, the pasted key, string, key, and array (containing an indented string) are indented inside the dict tag, spanning lines five through ten. The pasted content is at the same level of indentation as the rest of the keys and strings below it.">}}
+
+### Create the main file
+
+1. Open the <file>main.dart</file> file under the <file>lib/</file> directory.
+
+2. Replace the contents of this file with the following code to make up the login screen for your app:
```

Comment:
Nit: separate action from reason. `Replacing` doesn't technically itself `make`. Maybe something like:

```suggestion
2. Replace the contents of this file with the following code, which is responsible for the functionality of the login screen for your app:
```

Reply:
I mean in this case we are replacing a completely non-Viam-related demo screen with something useful, and the something does in fact include widgets and such which comprise the login screen. I worry that focusing only on the "functionality" implies that the login screen would be there anyway, but just wouldn't do anything. I think without this file change, it'd be a counter with a little button in the corner you can click to up the counter. The following code creates a non-functional button, so it actually doesn't create functionality but it does create a "Login" button that doesn't do anything. If the OG text is a no-go, is the following better?
```suggestion
2. Replace the contents of this file with the following code, which creates the scaffold of your app's login screen:
```